requirements.txt

```diff
@@ -1,6 +1,6 @@
 # regulations-core
 anyjson==0.3.3
-django==1.8.16
+django>=1.8,<1.9
```

Comment:
We generally want to pin our libraries to a specific version to ensure all environments match and that restaging results in the same app. This isn't an issue if you're vendoring all of your libraries, but 18F/openFEC#2271 implied that's not the current strategy.